{"name": "test_sync_phy_res", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: Element: xpath->//span[text()='DC1']/parent::div/parent::td/parent::tr/child::td/child::div/child::span/child::input not found in 5 seconds.", "trace": "self = <test_09_sync_phy_resource.TestPhySunc object at 0x01447DD0>\n\n    @pytest.mark.flaky(reruns=3)\n    def test_sync_phy_res(self):\n        datas = datainfo.get_xls_to_dict(\"user.xlsx\", \"Sheet1\")[0]\n        p_data = datainfo.get_xls_to_dict(\"res_node_data.xlsx\", \"region\")[0]\n    \n        self.login.login(datas[\"username\"], datas[\"password\"])\n        sync_a =  resSyncAction.ResSync(self.dr)\n        sync_pg = resSyncPage.ResSyncPage(self.dr)\n>       sync_a.phy_res_sync(p_data[\"regionname\"],p_data[\"nodename\"],\"DC1\",\"DC2\")\n\ntestcase\\test_09_sync_phy_resource.py:29: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\npublic\\appModel\\resSyncAction.py:39: in phy_res_sync\n    self.syncPage.select_and_click_data_center(DC)\npublic\\pages\\resSyncPage.py:39: in select_and_click_data_center\n    self.dr.click(\"xpath->//span[text()='%s']/parent::div/parent::td/parent::tr/child::td/child::div/child::span/child::input\"%value)\npublic\\common\\pyselenium.py:213: in click\n    self.element_wait(css)\npublic\\common\\pyselenium.py:91: in element_wait\n    WebDriverWait(self.driver, secs, 1).until(EC.presence_of_element_located((By.XPATH, value)), messages)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"5dd85394-4629-42a1-b106-b05950d20a67\")>\nmethod = <selenium.webdriver.support.expected_conditions.presence_of_element_located object at 0x01447F30>\nmessage = \"Element: xpath->//span[text()='DC1']/parent::div/parent::td/parent::tr/child::td/child::div/child::span/child::input not found in 5 seconds.\"\n\n    def until(self, method, message=''):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value is not False.\"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.time() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, 'screen', None)\n                stacktrace = getattr(exc, 'stacktrace', None)\n            time.sleep(self._poll)\n            if time.time() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: Element: xpath->//span[text()='DC1']/parent::div/parent::td/parent::tr/child::td/child::div/child::span/child::input not found in 5 seconds.\n\nD:\\Env\\ui_auto\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py:80: TimeoutException"}, "attachments": [{"name": "log", "source": "44000b6a-c36f-46eb-9d2f-306c2f3e851e-attachment.txt", "type": "text/plain"}], "start": 1572683666324, "stop": 1572683685545, "uuid": "66a68316-2c7b-479d-96c6-a08c2ca0edd8", "historyId": "bfc8f5de669fe7437eb149ff23841f95", "testCaseId": "b22921b8e1fb56764689e66f1e67f3fc", "fullName": "testcase.test_09_sync_phy_resource.TestPhySunc#test_sync_phy_res", "labels": [{"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_09_sync_phy_resource"}, {"name": "subSuite", "value": "TestPhySunc"}, {"name": "host", "value": "wuzs"}, {"name": "thread", "value": "5348-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_09_sync_phy_resource"}]}